package/MDAnalysis/analysis/helanal.py

```diff
@@ -243,7 +253,18 @@ def helanal_trajectory(universe, select="name CA",
 
     .. versionchanged:: 1.0.0
        Changed `selection` keyword to `select`
+
+    .. deprecated:: 1.0.1
+       helanal_trajectory is deprecated and will be removed in 2.0.0.
+       Please use helix_analysis.HELANAL instead (available in 2.0.0).
     """
+
+    warnings.warn("This function is deprecated as of MDAnalysis version 1.0. "
```

Comment:
I think we have this nice decorator for deprecations under lib.utils, you should be able to just do 

`@deprecate(release="1.0.1", remove="2.0", message="In 2.0 use the upcoming helix_analysis module.")`

for all these functions